### PR TITLE
Fix documentation tests.

### PR DESCRIPTION
**Pull request by baileyn**
Fixes the documentation tests so that `cargo test --doc` will succeed.

### CODE REVIEW

File: src/field.rs [hunk 1]
```diff
@@ -37,6 +37,6 @@ macro_rules! field {
     ($struct_name:ident, $field_name:ident) => {{
         let opt = None::<&$struct_name>.map(|v| &v.$field_name);
         let offs = $crate::__glium_offset_of!($struct_name, $field_name);
-        $crate::field::_hidden_field(offs, size, opt)
```
Comment (est31) [inline src/field.rs]: Could you maybe remove the size param entirely? It's not needed for anything.

Comment (baileyn) [reply src/field.rs]: I've went ahead and removed the size parameter :)

**Comment by est31** [comment]
Could you do a git rebase instead of merging my commits? The diff is messed up with the changes from the master branch.

**Comment by baileyn** [comment]
@est31 they're going through the pipeline now. Out of curiosity, does it always require a force push or am I doing something wrong?

**Comment by est31** [comment]
@baileyn yes a rebase always needs a force push. That's expected and okay. The goal is to have a clean history, which merge commits inside PRs don't give you.

**Pull request merged**